src/ui/server/controllers/insurance/your-buyer/traded-with-buyer/index.test.ts

```diff
@@ -66,7 +59,13 @@ describe('controllers/insurance/your-buyer/working-with-buyer', () => {
 
   describe('TEMPLATE', () => {
     it('should have the correct template defined', () => {
-      expect(TEMPLATE).toEqual(TEMPLATES.INSURANCE.YOUR_BUYER.WORKING_WITH_BUYER);
+      expect(TEMPLATE).toEqual(TEMPLATES.SHARED_PAGES.SINGLE_RADIO_PAGE);
```

Comment:
Should we rename `SINGLE_RADIO_PAGE` to just `SINGLE_RADIO`? Not other templates have "page" in the name?